app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<h2 class="form-heading">Sign up</h2>
+
+<div class="form-section flex justify-center my-10">
+  <div class="form-container border border-2 p-10">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+       <div class="form-group">      
+        <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: 'Username',  class: "form-control email_input" %>
+      </div>
+
+      <div class="field my-5">
+        
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email',  class: "form-control email_input" %>
+      </div>
+      
+      <div class="field my-5">
+        
+        <% if @minimum_password_length %>
+          <em>(<%= @minimum_password_length %> characters minimum)</em>
+        <% end %><br />
+        <%= f.password_field :password, autocomplete: "new-password", placeholder: 'Password',  class: "form-control email_input" %>
+      </div>
+
+      <div class="field my-5">
+    
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: 'Password', class: "form-control email_input" %>
+      </div>
+
+      <div class="actions my-5">
+        <%= f.submit "Sign up", class: "btn btn-primary"  %>
+      </div>
+
```

Comment:
- [ ] Dear Team, I was unable to create a user account once I set the project on my local machine, It keeps throwing an error about `unconfirmed_email`, and going back to the log I can confirm the user is saved to the database but I could not log in. I also try to find a confirmation token, but the app crashes before generating the token. Please address this one.

https://github.com/leloufadel/Recip-App/assets/69077061/3830cea4-21b3-4130-9f53-aa772bb7353e

- This also prevents me from checking the functionalities of the app, Please be reminded the next reviewer will be checking for the functionalities. Please make sure the functionalities are in place before the next review request.